packages/app/src/app/overmind/namespaces/files/internalActions.ts

```diff
@@ -99,7 +99,10 @@ export const uploadFiles: AsyncAction<
             file.type === 'application/json') &&
           dataURI.length < MAX_FILE_SIZE
         ) {
-          const text = dataURI ? atob(dataURI.replace(/^.*base64,/, '')) : '';
+          const text =
+            dataURI && dataURI !== 'data:'
```

Comment:
When the file is empty, `dataURI` is set to `data:`. Not exactly sure what the initial `dataURI` was supposed to check for, does it make sense to leave it in, or can we only do `dataURI !== 'data:'`? @CompuIves

Reply:
We can remove the `dataURI` check then I think, especially because there is a check on length before that already. Good find!

Reply:
Oh, I didn't see the suggestion! 🙂 Updated.